spark/spark-3.3/src/test/scala/org/apache/sedona/sql/OsmReaderTest.scala

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
```

Comment:
If the same OSM reader code works for all versions (i.e., in `spark/common`), we should move the test to `spark/common` as well.